find_number.rb

```diff
@@ -0,0 +1,41 @@
+# Program 3
+# Create find_number.rb.  You use the built-in Ruby random number generator.  The following returns a value from 1 to 100 inclusive:
+
+# secret_number = rand(1..100)
+# The program should do the following:
+
+# It generates a random number from 1 to 100.
+# In a loop, it prompts you for a guess. What you enter will be a string. So you convert it to an integer with the to_i method of the string class.
+loop do
+    secret_number = rand(1..100)

```

Comment:
With secret_number inside the loop it generates a new secret_number every time the user gives a response. We want the secret number to remain consistent  until guessed.

Reply:
Fixed!